broadinstitute/single_cell_portal_core
# Upgrade Ideogram to fix d3-color issue (SCP-4798)

This upgrades Ideogram.js by a minor version, fixing a [security issue in d3-color](https://github.com/broadinstitute/single_cell_portal_core/security/dependabot/55) reported by Dependabot.  It also coincidentally pulls in some upstream enhancements, but these don't affect any existing SCP UI or analytics.

To test:
* Pull branch, `yarn install --force`
* Run the following, and verify output matches below.
```
$ yarn why d3-color | grep 3.1.0
=> Found "d3-color@3.1.0"
```
* Open a realistic study, e.g. the "Human milk -- differential expression" DE pilot
* Search for a gene
* Confirm related genes ideogram appears

This satisfies SCP-4798.
